app/controllers/foods_controller.rb

```diff
@@ -42,31 +41,29 @@ def update
 
   def destroy
     food = Food.find(params[:id])
-    if @current_user.role === "admin" && food
+    if food
       food.destroy
       flash[:success] = "#{food.name} has been deleted."
-      redirect_to admin_foods_path
+      redirect_to dashboard_path
     else
       flash[:error] = "An error occured. Try deleting #{@food.name} again."
     end
   end
 
   def edit_status
-   if @current_user.role === "admin"
-     food = Food.find(food_params[:id].to_i)
-     if (!food.nil?)
-       status = food_params[:status] == "true" ? "available" : "not available"
-       food.status = status
-       food.save
-       render json: food
-     end
+   food = Food.find(food_params[:id].to_i)
+   if (!food.nil?)
```

Comment:
Favor `unless` over `if` for negative conditions.<br>Don't use parentheses around the condition of an `if`.